dev/com.ibm.ws.cdi.third.party_fat/test-applications/hibernateSearchTest.war/src/cdi/hibernate/test/web/SimpleTestServlet.java

```diff
@@ -49,23 +61,19 @@
 
     private static boolean fieldBridgeCalled = false;
     
-    @Override
-    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+    @Test
+    @Mode(TestMode.FULL)
+    @SkipForRepeat(NO_MODIFICATION)
+    public void testHibernateSearch()
             throws ServletException, IOException {
-        final PrintWriter pw = response.getWriter();
-        bean.getString();
-        pw.println("<h1>JPA Simple Test Case</h1>");
-        pw.println("<hr>");
         try {
-            // We trigger this method to indirectly call the field bridge.
-            CommonTestCode.populate(request, response, em, tx);
+            PrintWriter pw = new PrintWriter(new ByteArrayOutputStream());
+            CommonTestCode.populate(pw, em, tx);
```

Comment:
Do you want to check the contents of the PrintWriter after the populate call? A PrintWriter probably isn't the best way to do it any more anyway?